packages/typespec-azure-portal-core/src/decorators.ts

```diff
@@ -0,0 +1,364 @@
+import { ArmResourceKind, getArmResourceKind } from "@azure-tools/typespec-azure-resource-manager";
+import {
+  CompilerHost,
+  DecoratorContext,
+  Model,
+  ModelProperty,
+  Program,
+  StringLiteral,
+  Type,
+  getDirectoryPath,
+  getSourceLocation,
+  normalizePath,
+  resolvePath,
+  validateDecoratorUniqueOnNode,
+} from "@typespec/compiler";
+import * as fs from "fs";
+import { PortalCoreKeys } from "./keys.js";
+import { reportDiagnostic } from "./lib.js";
+import { AboutOptions, BrowseOptions, marketplaceOfferOptions } from "./types.js";
+
+/**
+ * This is a Browse decorator which will be use to put more info on the browse view.
+ * @param target The model that is being decorated.
+ * @param options BrowseOptions of the property.
+ */
+export async function $browse(context: DecoratorContext, target: Model, options: BrowseOptions) {
```

Comment:
```suggestion
export async function $browse(context: DecoratorContext, target: Model, options: Model) {
```

keep it as the actual type it is as its not a `BrowseOptions` object

Reply:
fixed for BrowseOptions AboutOptions and MarketplaceOfferOptions :) thank you